packages/data-stores/src/automated-transfer-eligibility/index.ts

```diff
@@ -1,25 +1,22 @@
-import { registerStore } from '@wordpress/data';
+import { register, createReduxStore } from '@wordpress/data';
 import * as actions from './actions';
 import { STORE_KEY } from './constants';
 import reducer from './reducer';
 import * as resolvers from './resolvers';
-import * as selectors from './selectors';
 import { State } from './types';
+export * from './utilities';
 
 export * from './types';
 export type { State };
 export { STORE_KEY };
 
-let isRegistered = false;
-export function register(): typeof STORE_KEY {
-	if ( ! isRegistered ) {
-		isRegistered = true;
-		registerStore( STORE_KEY, {
-			actions,
-			reducer,
-			resolvers,
-			selectors,
-		} );
-	}
-	return STORE_KEY;
-}
+export const store = createReduxStore( STORE_KEY, {
+	actions,
+	reducer,
+	resolvers,
+	selectors: {
+		getAutomatedTransferEligibility: ( state: State, siteId: number | null ) =>
+			siteId ? state[ siteId ] : null,
```

Comment:
Could theoretically this return `undefined` if `state` doesn't have a `siteId` key yet? Do we want to return `null` in that scenario?

Reply:
Good point, I'll update that.